code/obj/item/storage/bible.dm

```diff
@@ -107,8 +115,12 @@
 
 			else
 				var/damage = 10 - clamp(target.get_melee_protection("head", DAMAGE_BLUNT) - 1, 0, 10)
+				var/faith_gain = damage
 				if (is_atheist)
 					damage /= 2
+					faith_gain *= 2 // hitting atheists gives more
+
+				add_faith(faith_gain * FAITH_HEAL_FAIL_GAIN)
```

Comment:
might encourage a bit more violence than expected?

Reply:
nvm i see it's only if the heal fails

Reply:
Healing atheists will always fail so your point is valid. The number is supposed to be small enough that it's more of a joke than a legitimate thing but it still might